Engine/Module/ModuleScene.cpp

```diff
@@ -384,8 +390,16 @@ inline void ModuleScene::LoadSceneResource()
 	}
 	else
 	{
+		if(App->time->isGameRunning())
+		{
+			timer.Start();
+			LoadLoadingScreen();
+		}
```

Comment:
Check here if `#MULTITHREADING` is needed

Reply:
Done